components/data-table.d.ts

```diff
@@ -62,6 +62,14 @@ declare module '@salesforce/design-system-react/components/data-table' {
 		 * Use this if you are creating an advanced table (selectable, sortable, or resizable rows). Columns widths will be truncate based on width and DOM ancestors. See `fixedHeader` to enable horizontal and vertical scrolling.
 		 */
 		fixedLayout?: boolean;
+		/**
+		 * When fixedHeader is true, specifies whether there's more data to be loaded and displays a spinner at the bottom of the table if so.
+		 */
+		hasMore?: boolean,
```

Comment:
It looked like this prop was missed when the TypeScript typings were created.

Reply:
The types were added, but they are automatically generated at this time. Please call them out when you find them.